runtime/v2/shim/shim.go

```diff
@@ -155,6 +158,11 @@ func Run(id string, initFunc Init, opts ...BinaryOpts) {
 
 func run(id string, initFunc Init, config Config) error {
 	parseFlags()
+	if versionFlag {
+		fmt.Fprintln(os.Stdout, version.Version)
```

Comment:
Can we print both containerd version (v1.4.0) and shim API version (v2)?

Reply:
In my draft version of this, I copied it from `containerd`
```
               fmt.Println("containerd-shim:")
               fmt.Println("  Version: ", version.Version)
               fmt.Println("  Revision:", version.Revision)
               fmt.Println("  Go version:", version.GoVersion)
               fmt.Println("")
```

Reply:
thanks, I updated the output based on your suggestion:

```
containerd-shim-runc-v2:
  Version:  v1.3.0-412-g939158a6.m
  Revision: 939158a6d7ccf28af6f919e2383a31e25b550eb9.m
  Go version: go1.13.4
```